### PR TITLE
Adding Changelog for the Datadog Cluster Agent 1.0.0

### PR DESCRIPTION
**Pull request by CharlyF**
### What does this PR do?

Adding the Changelog for the 1.0.0 of the Datadog Cluster Agent.

### CODE REVIEW

File: CHANGELOG-DCA.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+=====
+
+.. _Release Notes_1.0.0_Prelude:
+
+Prelude
+-------
+
+Released on: 2018-10-18
+
+The Datadog Cluster Agent is compatible with the Datadog Agent 6.5.0.
```
Comment (xvello) [inline CHANGELOG-DCA.rst]: Forward-compat is a given for us, but being explicit is probably better. WDYT of something along the lines of `compatible with versions 6.5.1 and up of the Datadog Agent`

File: CHANGELOG-DCA.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+- Please find the image on `our Docker Hub <https://hub.docker.com/r/datadog/cluster-agent/tags/>`_.
+
+1.0.0-rc.4
+==========
+2018-10-17
+
+Enhancement Notes
+------------------
+- Expose telemetry metrics with the Open Metrics format instead of goexpvar
```
Comment (mfpierre) [inline CHANGELOG-DCA.rst]:
```suggestion
- Expose telemetry metrics with the Open Metrics format instead of expvar
```

**Review by mfpierre — COMMENTED** [review]
Should we seize the opportunity to add the dedicated cluster agent reno folder?

**Review by mfpierre — APPROVED** [review]

**Comment by CharlyF** [comment]
appveyor failure due to a flaky unit test.

**Pull request merged**